instruqt/protecting-your-assets/01-the-replicated-license/assignment.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: the-replicated-license
+id: 4udtdfaiyfyr
+type: challenge
+title: The Replicated License
+teaser: Understanding the role of the Replicated license
+notes:
+- type: text
+  contents: Understanding the Replicated license
+tabs:
+- title: License File
+  type: code
+  hostname: shell
+  path: /home/replicant
+difficulty: basic
+timelimit: 400
+---
+
+👋 Introduction
+===============
+
+The Replicated Platform provides a license that identifies each of your
+customers and entitles them to install your software. These customer licenses
+provide several default entitlements such as the type of licesnse and whether
+(and when) it expries. Replicated provides the license file to you as a
+downloadable asset and embeds it into your Helm chart when you include the
+Replicated SDK. A digital signature confirms it hasn't been tampered with.
+
+How License Work
+================
+
+Each customer you create for your application in the Replicated Vendor Portal
+has a license associated with it. The main purpose of the license is to
+identify which software the user is entitled to. The entitlement is determined
+by two attribute of the license: the application and the release channel. The
+platform knows the application the customer was created for, and you specify
+the release channel when you create the customer. You can change the channel at
+any time. We'll discuss release channels shortly.
+
+The other core attribute of the license is it's type. Each customer license has
+one of the following types:
+
+* **Development:** The Development type can be used internally by the
+  development team for testing and integration.
+* **Trial:** The Trial type can be used for customers who are on 2-4 week
+  trials of your software.
+* **Paid:** The Paid type identifies the customer as a paying customer for
+  which additional information can be provided.
+* **Community:** The Community type is designed for a free or low cost version
+  of your application. For more details about this type, see Community
+  Licenses.
+
+You can change the license type of an existing license. You'll most often do
```

Comment:
Good question. I've certainly moved them around up and down in the vendor portal, but I think KOTS has limitations as to which will sync.

Reply:
@brianrhea @AmberAlston I know I've heard about limitations here, but I definitely change my licenses back and forth among types all the time. Is the limitation in KOTS rather than the vendor portal?

Reply:
Good Slack discussion on this on this morning, and I also checked in with @grantmiller. Switching types is allowed and intended to be. Changing a license in a KOTS install is restricted. But may not be intended to be.